master/internal/command/command.go

```diff
@@ -424,7 +424,7 @@ func (c *command) toV1Job() *jobv1.Job {
 		Name:           c.Config.Description,
 	}
 
-	j.IsPreemptible = config.ReadPreemptionStatus(j.ResourcePool, &c.Config)
+	j.IsPreemptible = config.ReadRMPreemptionStatus(j.ResourcePool) && false
```

Comment:
this is always false?

Reply:
ture. I wanted to keep this similar to how IsPreemtible is calculated on Experiment jobs but it does seem wasteful. I wrote down more on the other one. should we avoid throwing a `false` here without it being linked to anywhere else